src/hashspace.rs

```diff
@@ -230,12 +228,12 @@ impl Hashspace {
   }
 
   #[cfg(not(target_arch = "wasm32"))]
-  pub fn put_local_storage(&self, _cid: String, _data: &str) -> Result<(),String> {
+  pub fn put_local_storage(&self, _cid: &str, _data: &str) -> Result<(),String> {
```

Comment:
We shouldn't change the type signatures of stubbed out functions that @Anderssorby is working on

Reply:
Sure, okay. This one and the one next to it?